### ISSUE TITLE
Text rendering and context zoom

### ISSUE THREAD

**Issue by HenryTheCat** [PistonDevelopers/piston#1201]
While experimenting with Piston, I used `piston_window` and `piston_graphics` to draw some graphics.

To make the content adapt to the size of the window, I normalized the `context` via `context.transform.zoom` to a chosen dimension, so that that reference dimension would always be `1`.

Generally, that worked alright, until I tried to draw some text on the screen. The size of the text has to be an integer, so `1` is the smallest possible, but in this case `1` corresponds to a far too big size on screen, due to the normalization.

Also, the text is rasterized according to its size, so if I pick size `1` the text will be rasterized in low resolution; then, it gets scaled, and the result is horrible.

As a workaround, I changed the normalization default to, say, `100`, so that a font size of `10` looks good. This is a bit hacky, though, and the choice of the default is tricky: if I use too a high value, I get overflow errors.

Long story short, text size doesn't work nice with scaling, unlike textures. Is that a defect, or is there something wrong with my approach?

(I hope the description of the issue is clear enough. I can elaborate more if necessary.)

**Comment by bvssvni** [comment]
You should use a larger text size and then zoom back the transformation when rendering the text.

**Comment by HenryTheCat** [comment]
It seems not ideal to me to force users to rescale twice to write text, but if that's the intended behaviour I'll close the issue.

**Comment by bvssvni** [comment]
You're zooming in with the coordinate system, but the text is rasterized in the points coordinate system (twice the pixels on retina screen). The only other option would be to rasterize on the GPU, but Piston does not have this feature yet.
